### PR TITLE
fix: alignment of name for searched profiles

### LINKED ISSUE #4142: Bug: when a user's name is short, it aligns horizontally with the profile picture in the search results
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

I've noticed a bug where, in the search results, if a user's name is short, their name aligns horizontally with the profile picture. I believe this alignment should be corrected to ensure a better user experience.

### Steps To Reproduce

1. go to https://circuitverse.org.
2. Use the search bar to look for users.
3. Observe users with short names in the search results.

### Expected Behavior

regardless of the length of a user's name, the alignment of their name and the profile picture should always be visually consistent and correctly formatted.

### Screenshots

Screenshot 2023-10-07 at 11 53 27 AM

### Device Information [optional]

```markdown
- OS: MacOs Sonoma
- Browser: Safari
- version: 16.5
```

### Additional context

_No response_

### Are you working on this issue?

Yes

### PR DESCRIPTION
**Pull request by niladrix719**
Fixes #4142

#### Describe the changes you have made in this PR -

removed a div which wraps the user name and profile picture as a row

### Screenshots of the changes (If any) -

Screenshot 2023-10-07 at 3 07 49 PM

### CODE REVIEW

**Comment by niladrix719** [comment]
cc @vedant-jain03

File: app/views/users/circuitverse/search.html.erb [hunk 1]
```diff
@@ -14,10 +14,8 @@
     <% @results.map { |user| ProfileDecorator.new(user) }.each do |profile| %>
       <div class="row center-row search-user-container">
         <div class="col-xs-12 col-sm-12 col-md-5 col-lg-5">
-          <div class="row center-row">
-            <%= image_tag user_profile_picture(profile.profile_picture), alt: "project.name", class: "center-row search-usersearch-image" %>
-            <p class=" center-row search-username"><%= profile.name %></p>
-          </div>
+          <%= image_tag user_profile_picture(profile.profile_picture), alt: "project.name", class: "center-row search-usersearch-image" %>
+          <p class=" center-row search-username"><%= profile.name %></p>
```
Comment (vedant-jain03) [inline app/views/users/circuitverse/search.html.erb:18]: Better to place under div, we can give class; `flex flex-col` so that it appears in expected way

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]

**Comment by niladrix719** [comment]
cc @vedant-jain03

**Comment by niladrix719** [comment]
cc @tanmoysrt

**Review by tanmoysrt — APPROVED** [review]
Working as expected!

**Review by vedant-jain03 — APPROVED** [review]

**Pull request merged**